docs/how-to-build.md

```diff
@@ -1,51 +1,317 @@
+```
+bash> sudo yum -y install autoconf automake libtool cmake bison unzip boost gperf krb5 openssl libunwind ncurses readline
+```
+
+#### Build and install 3rd-party Libraries
+The third-party was installed to **/opt/nebula/third-party**
+
+```
+bash> git clone git@github.com:vesoft-inc/nebula-3rdparty.git OR git clone https://github.com/vesoft-inc/nebula-3rdparty.git
+bash> cd nebula-3rdparty
+bash> cmake ./
+bash> cmake -DSKIP_JAVE_JAR=OFF  ./  # if not need to build jave client
+bash> make
+bash> sudo make install
+```
+
+#### Build nebula
+The default installation dir is **/usr/local/nebula**
+
+```
+bash> git clone git@github.com:vesoft-inc/nebula.git OR git clone https://github.com/vesoft-inc/nebula.git
+bash> cd nebula && mkdir build && cd build
+bash> cmake ..
+bash> cmake -DSKIP_JAVE_CLENT=ON ..  # if not need to build jave client
+bash> make OR make -j${threadnum}
+bash> sudo make install
+```
+
+### How to build in Centos7.5
+
+#### Prepare
+**Step1**: install tools
+
+```
+bash> sudo yum -y install git git-lfs
+```
+**Step2**: install dependencies
+
+through yum install
+
+```
+bash> sudo yum install -y libtool autoconf autoconf-archive automake perl-WWW-Curl libstdc++-static maven
+```
+
+through vesoft offer
+
+```
+bash> wget https://nebula-graph.oss-cn-hangzhou.aliyuncs.com/build-deb/centos-7.5-1804.tar.gz
+```
+
+1) Create a system user, which home directory will be used as a shared file location
+
+```
+bash> sudo adduser --system --group --home /home/engshare engshare
+```
+
+Please make sure the user's home directory **/home/engshare** is readable to all users
+
+2) Install all necessary rpm packages in this folder
+
+```
+bash> tar xf centos-7.5-1804.tar.gz && cd centos-7.5-1804/
+bash> rpm -ivh *.rpm
+```
+
+3) Modify **~/.bashrc** by appending following lines to the end
+
+```
+alias cmake='/home/engshare/cmake/bin/cmake -DCMAKE_C_COMPILER=/home/engshare/gcc/bin/gcc -DCMAKE_CXX_COMPILER=/home/engshare/gcc/bin/g++ -DNEBULA_GPERF_BIN_DIR=/home/engshare/gperf/bin -DNEBULA_FLEX_ROOT=/home/engshare/flex -DNEBULA_BOOST_ROOT=/home/engshare/boost -DNEBULA_OPENSSL_ROOT=/home/engshare/openssl -DNEBULA_KRB5_ROOT=/home/engshare/krb5 -DNEBULA_LIBUNWIND_ROOT=/home/engshare/libunwind'
+
+alias ctest='/home/engshare/cmake/bin/ctest'
+```
+4) Update **~/.bashrc**
+
+```
+bash> source ~/.bashrc
+```
+
+#### Build and install 3rd-party Libraries
+The third-party was installed to **/opt/nebula/third-party**
+
+```
+bash> git clone git@github.com:vesoft-inc/nebula-3rdparty.git OR git clone https://github.com/vesoft-inc/nebula-3rdparty.git
+bash> cd nebula-3rdparty
+bash> cmake ./
+bash> cmake -DSKIP_JAVE_JAR=OFF  ./  # if not need to build jave client
+bash> make
+bash> sudo make install
+```
+
+#### Build nebula
+The default installation dir is **/usr/local/nebula**
+
+```
+bash> git clone git@github.com:vesoft-inc/nebula.git OR git clone https://github.com/vesoft-inc/nebula.git
+bash> cd nebula && mkdir build && cd build
+bash> cmake ..
```

Comment:
.. or ./ ?

Reply:
.. and ./ bases on the file path is not the same, please read the previous command carefully.